pkg/controllers/sidecar/config/config.go

```diff
@@ -156,6 +158,22 @@ func loadProperties(fileName string, logger log.Logger) *properties.Properties {
 	return p
 }
 
+var scyllaArgumentsRegexp = regexp.MustCompile(`--([^= ]+)(="[^"]+"|=[^ ]+|[ \t]+"[^"]+"|[ \t]+[^-][^-]?[^ ]*|)`)
+
+func convertScyllaArguments(scyllaArguments string) map[string]*string {
+	output := make(map[string]*string)
```

Comment:
nil pointers aren't added to this map, you can keep `map[string]string`. Later there's unnecessary dereferece:  `pointer2.StringPtr(strings.TrimSpace(*argValue))`

Reply:
fixed